app/atoms/base/_media.scss

```diff
@@ -0,0 +1,7 @@
+/* media queries */
+$mobile_xs: new-breakpoint(max-width 319px 12);
+$mobile_s: new-breakpoint(min-width 320px 12);
+$mobile_l: new-breakpoint(min-width 480px 12);
+$tablet: new-breakpoint(min-width 768px 12);
+$desktop: new-breakpoint(min-width 960px 12);
+
```

Comment:
@adamf321 we need add the $container  breakpoint, with the width of the container